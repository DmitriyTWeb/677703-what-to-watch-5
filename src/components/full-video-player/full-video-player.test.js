import React, {createRef} from "react";
import renderer from "react-test-renderer";
import FullVideoPlayer from "./full-video-player";

const noop = () => { };


it(`Should FullVideoPlayer render correctly`, () => {
  const tree = renderer.create(
      <FullVideoPlayer
        src={``}
        fullImg={``}
        onVideoMount={noop}
        videoRef={createRef()}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

