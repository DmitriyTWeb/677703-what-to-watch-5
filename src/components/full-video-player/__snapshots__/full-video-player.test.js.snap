// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FullVideoPlayer render correctly 1`] = `
<video
  className="player__video"
  poster=""
  src=""
>
  Please use another web browser with video tag support
</video>
`;
